frontend/src/scenes/persons/Cohorts.tsx

```diff
@@ -85,6 +85,9 @@ function _Cohorts(): JSX.Element {
                 </span>
             ),
             render: function RenderCalculation(_, cohort: CohortType) {
+                if (cohort.is_static) {
+                    return 'N/A'
```

Comment:
This is causing a ts error since it isn't returning a JSX.Element -> Wrap it in a fragment. :)